src/bibliography/ui/ReferenceList.tsx

```diff
@@ -28,13 +28,11 @@ export default function ReferenceList({
 }): JSX.Element {
   return (
     <>
-      {_.isEmpty(references) ? (
-        <p>No references</p>
-      ) : (
-        groupReferences(references).map(([type, group]) => (
-          <ReferenceGroup key={type} references={group} />
-        ))
-      )}
+      {_.isEmpty(references)
+        ? null
+        : groupReferences(references).map(([type, group]) => (
```

Comment:
In the Fragment Editor "No References" won't be shown because the component is used in two places (Editor and SearchResults)